tests/linux_kernel/helpers/test_slab.py

```diff
@@ -75,6 +76,31 @@ def test_slab_cache_is_merged_true(self):
             self.fail("couldn't find slab cache")
         self.assertTrue(slab_cache_is_merged(slab_cache))
 
+    def test_slab_cache_aliases(self):
+        alias_to_name = get_slab_cache_aliases(self.prog)
```

Comment:
This will blow up on kernels with SLAB or SLOB. We have kernels with SLAB and SLOB built, we just don't run them in the CI because it'd take forever. You can test both manually with:

```
python3 setup.py test -k '*alternative,*tiny'
```

If `/sys/kernel/slab` doesn't exist, we should check that `get_slab_cache_aliases()` raises your exception (e.g., with `self.assertRaisesRegex()`).